chrome/content/zotero/xpcom/data/feedItems.js

```diff
@@ -40,6 +40,7 @@ Zotero.FeedItems = new Proxy(function() {
 			let obj = zi_primaryDataSQLParts.call(this);
 			obj.feedItemGUID = "FeI.guid AS feedItemGUID";
 			obj.feedItemReadTime = "FeI.readTime AS feedItemReadTime";
+			obj.feedItemTranslatedTime = "FeI.translatedTime AS feedItemTranslatedTime";
```

Comment:
Unless `FeI` is actually necessary for disambiguation, these should just be `FI`, as we do for other two-word tables.